common/lib/xmodule/xmodule/course_module.py

```diff
@@ -682,7 +682,7 @@ class CourseFields(object):
         ),
         scope=Scope.settings,
         # Ensure that courses imported from XML keep their image
-        default="images_course_image.jpg"
+        default=""
```

Comment:
Apparently this note was not explicit enough. This is what I think, but I am not on the TNL team. As mentioned below, I am not qualified to really review this PR and someone in this code area needs to weigh in on this change, and whether or not it is the right way to go.